controllers/configauditreport/configauditreport_controller.go

```diff
@@ -101,7 +102,7 @@ func (r *ConfigAuditReportReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		}
 	}
 
-	return defaultRequeue(), nil
+	return utils.JitterRequeue((time.Minute * 5), r.MaxJitterPercent, r.Log), nil
```

Comment:
I feel like we are spreading the `5 minute` time into multiple files now. Can we define this as a constant somewhere?